1.8/administration/access-node/tunnel.md

```diff
@@ -0,0 +1,116 @@
+---
+nav_title: Access by Proxy
+post_title: Access by Proxy using DC/OS Tunnel
+menu_order: 10
+---
+
+When developing applications on DC/OS, you may find it helpful to access your cluster from your local machine via SOCKS proxy, HTTP proxy, or VPN. For instance, you can work from your own development environment and immediately test against your DC/OS cluster.
+
+# SOCKS
+DC/OS Tunnel can run a SOCKS proxy over SSH to the cluster. SOCKS proxies work for any protocol, but your application must be configured to use the proxy, which runs on port 1080 by default.
+
+# HTTP
+Run the HTTP proxy on your local port 80 to tunnel HTTP requests without modifying your application. Access URLs by appending the `mydcos.directory` domain. You can also [use DNS SRV records as if they were URLs](#srv). The HTTP proxy requires root access to use port 80, but can be configured to use a different port.
+
+<a name="srv"></a>
+## SRV Records
+A SRV DNS record is a mapping from a name to a IP/port pair. DC/OS creates SRV records in the form `_<port-name>._<service-name>._tcp.marathon.mesos`. The HTTP proxy exposes these as URLs. This feature can be useful for communicating with DC/OS services.
+
+# VPN
+DC/OS Tunnel provides you with full access to the DNS, masters, and agents from within the cluster. OpenVPN requires root privileges to configure these routes.
+
+# Using DC/OS Tunnel
+
+## Prerequisites
+* DC/OS 1.8+
+* Docker
```

Comment:
In that case, we could remove the prereqs entirely, since these are the DC/OS docs. Having the CLI installed is a prerequisite, though. I'll add that and remove everything else.